release_notes.md

```diff
@@ -1,59 +1,16 @@
-  - Block or enrich indicators
-  - Escalate alerts to an incident
-  - Create a war room for a given incident
-  - Create or update FortiSOAR module records
-  - Correlate records
-  - Generate a FortiSOAR playbook
-  - Create Jinja expressions and Regex to speed up playbook development in FortiSOAR
-  - Navigate to a FortiSOAR module such as Alerts, Incidents, or Indicators
-  - Navigate to a specific record in a FortiSOAR module
+**Conversation Thread Management**
 
-- The bot asks questions if it needs more information or context to perform a task. For example, when blocking an indicator, it requests the indicator's value, type, and reason for blocking.
+Previously, conversation threads were deleted from OpenAI whenever *Clear Conversation* was clicked, and a new thread was started with each new prompt.
 
->[!Note]
->The bot honors RBAC (Role-based Access Control) settings are when executing tasks requested in the conversation.
-
-- The bot is not limited to the aforementioned SOC tasks and can assist in investigations by asking cybersecurity questions.
-
-- The bot holds conversations to understand requirements and generate a more targeted playbook. The experience is streamlined, allowing users to engage with the bot during the creation of a playbook outline, and making it easier to modify the outline by adding, updating, or removing steps.
-
-- Previous conversations remain in the conversation window and provide context to the bot until users clear the conversation. Conversations persist throughout repeated logins and logouts.
-
-- With the bot's enhanced understanding of natural language, it derives its operating modes from the conversation, making the Playbook Blocks and Conversation Mode, unnecessary.
-
->[!Note]
->Sensitive data is still actively masked in all conversations with the LLM.
-
-- Updated the conversation and playbook models to the following:
-  - `gpt-4o-mini-2024-07-18`
-  - `gpt-4o-mini`
-  - `gpt-4o-2024-05-13`
-  - `gpt-4-turbo-2024-04-09`
-
-- The playbook collection *10 - SP - Fortinet Advisor* has been renamed to **10 - SP - FortiAI**.
-
-- The following playbooks have been deprecated, as these actions can be performed by the bot without triggering any of these playbooks:
-  - Converse with LLM
-  - Generate WorkPlan for User Inputs
-  - Get Alert Summary
-  - Get Incident Summary
-  - Get Alert Response Plan
-  - Get Incident Response Plan
-  - Get Alert Report
-  - Get Incident Report
-  - Get Alert MITRE Insight
-  - Get Incident MITRE Insight
-
->[!Note]
->Delete these playbooks manually when upgrading from an older **FortiAI** version.
\ No newline at end of file
+Now, threads are retained within OpenAI, enabling improved historical reference and continuity. Each new prompt still initiates a fresh thread, while the user interface continues to clear conversation history after each prompt, ensuring a clean workspace.
```

Comment:
This sounds confusing! Need to rephrase this.

Reply:
Changed